build.xml

```diff
@@ -40,6 +40,7 @@
     <property name="projectHome" location="." />
 
 	<target name="buildjar">
+		<mkdir dir="${projectHome}/bin"/>
```

Comment:
This should be its own commit as its unrelated to the Project name change.

Reply:
fixed